stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -353,6 +353,13 @@ module SwiftGlibc [system] {
       }
 % end
 
+% if CMAKE_SDK in ["LINUX"]:
+    module xattr {
+        header "${GLIBC_ARCH_INCLUDE_PATH}/sys/xattr.h"
```

Comment:
Hmm, I don't see this include available for me.  Are you sure that this shouldn't be `${GLIBC_INCLUDE_PATH}/sys/xattr.h`?